src/wallet/wallet.cpp

```diff
@@ -3513,6 +3513,26 @@ bool CWallet::GetKeyFromPool(CPubKey& result, bool internal)
     return true;
 }
 
+bool CWallet::GetNewDestination(const OutputType type, const std::string label, CTxDestination& dest, std::string& error)
+{
```

Comment:
micro-nit: can clear `error` here.

Reply:
Does it matter?

Reply:
micro-matters in case someone calls the function twice and doesn't clear the error themself? :)

Reply:
Done